UI/js/signinNavigations.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+
+let user;
+let statusDiv;
+
+function initPage(url) {
+  statusDiv = document.getElementById('status');
+
+  user = getParameterByName('user', url);
+  let action = getParameterByName('action', url);
+  user = user === 'engineer' ? user : 'user';
+
+  action = action === 'signup' || action === 'reset' ? action : 'signin';
+
+
+  document.body.className = `${user} ${action}`;
+
+  Array.from(document.querySelectorAll('a[href]'))
```

Comment:
`document.querySelectorAll` will automatically return multiple elements that you can treat as an array. Why do you need to use the `Array` method?

Reply:
Thanks. I would look at that one.